tests/tfengine/integration_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Google LLC
```

Comment:
There is no notation that I am aware of that restricts all code running inside a test to be written in go. I added a prerequisites section on top to indicate the terraform bin must be available. It will fail anyway if the binary isn't there so there isn't much accidental harm that can be caused by this.

Reply:
What do you think about changing it from a "test" (with t *testing.T) to just a regular binary that can be run on purpose, but isn't caught by `go test ./...`?

Reply:
I think it is more useful for this test to be runnable as part of go test ./... and I think this should be run by all users at a minimum locally before sending any PRs. It will fail if the tf binary is not available with a pretty clear message and we have given a description of prerequisites now so I don't think the additional complexity of needing an extra binary to run the tests outweighs the benefit of running it easily as part of the rest of the test suite.

I also prefer reducing the number of shell scripts in our codebase, I personally find it much more maintainable long term to write go :)